datasets/crucial/evaporation_deficit_upload.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+from os.path import basename
+from os import environ
+import subprocess
+from datetime import datetime
+import glob
+
+from google.cloud import storage
+
+def upload_blob(bucket_name, source_file_name, destination_blob_name):
```

Comment:
These upload functions are already in some other datasets, couldn't we re-use them?

Reply:
This whole file could be replaced by a small README.md that explains:
- Source of data (P disk/project something/where to the .nc files come from)
- GEE upload path

Reply:
Removes unneeded print statements and commented out lines. Regarding re-using upload functions already in use for other datasets: could (re)use utils.py as included in glossis folder for this, but might be useful to reorganize some of the files in \datasets in that case (move utils.py up one level?). Would suggest not to do this now, but to discuss how to best organize this first, and do in the next sprint where this functionality is relevant.